plaid/institutions.go

```diff
@@ -13,6 +14,32 @@ type Institution struct {
 	Name         string       `json:"name"`
 	Products     []string     `json:"products"`
 	CountryCodes []string     `json:"country_codes"`
+
+	// Included when `options.include_status` is true.
+	InstitutionStatus InstitutionStatus `json:"status"`
```

Comment:
if its nullable, should it be a pointer` ? I think thats the more idiomatic thing to do for structs.

Reply:
sure, but lmk know what you think after the update. primary colour and logo may be empty string which keeps as null anyway.